fortuna/src/models/casusBlockModel.js

```diff
@@ -4,7 +4,7 @@
 // Required Imports
 import Mongoose from 'mongoose';
 
-const CasusBlock = new Mongoose.Model('CasusBlock', new Mongoose.Schema({
+const CasusBlock = new Mongoose.model('CasusBlock', new Mongoose.Schema({
```

Comment:
I'm sure this is necessary, but I just find it curious that it worked before when it was lowercase...

Reply:
To be honest, I actually never confirmed if it worked correctly. So it's good that it caught it here.